src/toast/spt3g/spt3g_utils.py

```diff
@@ -227,7 +223,9 @@ def compress_timestream(ts, params, rmstarget=2**10, rmsmode="white"):
         offset (float) :  The removed offset
 
     """
-    if not params:
+    if params is None:
+        return ts, 1, 0
+    if isinstance(params, bool) and not params:
```

Comment:
seems like this test could have been `if params is None or params is False` but just as fine like this.

Reply:
Fixed